src/lib/Runner.ts

```diff
@@ -174,12 +174,12 @@ export class Runner {
     return state.get();
   };
 
-  public isPaused = async () => {
+  public isPaused = async (): Promise<string | false> => {
```

Comment:
`string | false` makes it hard to do a "falsey" check which can lead to bad assumption.  (Empty strings are falsey).

If you want to get the reason out of this method the return type should be.

```typescript
public getPauseState = async (): Promise<{
  isPaused: true;
  reason: string;
} | {
  isPaused: false;
}> => {}
```